Test/QuickCheck/Modifiers.hs

```diff
@@ -343,4 +355,100 @@ instance (Arbitrary a, ShrinkState s a) => Arbitrary (Shrinking s a) where
 #endif /* NO_MULTI_PARAM_TYPE_CLASSES */
 
 --------------------------------------------------------------------------
+-- | @ASCII@.
+newtype ASCII = ASCII {getASCII :: Char}
+  deriving ( Eq, Ord, Show, Read )
+
+asciiGen :: Gen Char
+asciiGen = choose ('\0', '\127')
+
+instance Arbitrary ASCII where
+  arbitrary = ASCII `fmap` asciiGen
+
+--------------------------------------------------------------------------
+-- | @Latin1@: latin1 Char, same Arbitrary as Char itself.
+newtype Latin1 = Latin1 {getLatin1 :: Char}
```

Comment:
Yeah, ideally I'd change the `Arbitrary Char` instance to return unicode Chars... in the meanwhile I'll add an haddock

not sure how `-----` affects it, but it was already present for every other newtype